core/lib/dal/migrations/20250425113231_drop_ix_initial_writes_t1.up.sql

```diff
@@ -0,0 +1 @@
+DROP INDEX ix_initial_writes_t1;
```

Comment:
Maybe

```suggestion
DROP INDEX IF EXISTS ix_initial_writes_t1;
```

and run it manually on mainnet when it's convinient for us to monitor the impact?